### PR TITLE
Add basic API tutorials

### PR DESCRIPTION
**Pull request by shanamatthews**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Basic tutorial for using the [List an Organization's Teams](https://docs.sentry.io/api/teams/list-an-organizations-teams/) and [Create a New Team](https://docs.sentry.io/api/teams/create-a-new-team/) endpoints.

Links:
- New [Create an Auth Token](https://sentry-docs-git-shana-api-tutorials.sentry.dev/api/guides/create-auth-token/) tutorial
- New [Create and List Teams](https://sentry-docs-git-shana-api-tutorials.sentry.dev/api/guides/teams-tutorial/) tutorial

"Create and Configure a Project with the Sentry API" guide is currently an outline and not ready for review

Todo:
- [x] Add screenshots/Arcades
- [x] Split get token into separate tutorial & link??
- [x] Make sidebar section for tutorials
- [ ] ~~how to duplicate a project~~
- [ ] ~~add in placeholders for endpoints that don't exist~~

### CODE REVIEW

**Comment by sentaur-athena** [comment]
@shanamatthews I can't see the links to your dev env, is it possible to access it or include screenshots here to review?

File: src/api/tutorials/create-auth-token.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+- A Sentry account with an organization-level role of Manager or Admin.
+
+## Create an Internal Integration
+
+[Internal integrations](/product/integrations/integration-platform/internal-integration/) are used to create custom Sentry integrations for your organization. They can also be used to create and manage your organization auth tokens.
+
+1. Open [sentry.io](https://sentry.io/)
+
+1. Click "Settings" in the left menu to open the **Organization Settings** page.
```
Comment (sentaur-athena) [inline src/api/tutorials/create-auth-token.mdx:20]: Why are these all 1?

**Comment by shanamatthews** [comment]
> @shanamatthews I can't see the links to your dev env, is it possible to access it or include screenshots here to review?

Hmm, you need Vercel access. I thought folks got it by default but maybe not. 
Try this?

### Vercel Access

To join Vercel, visit this link: http://vercel.com/teams/invite/Y3EF7uU8.

### Preview Builds

Vercel allows you to view preview builds of your branch when you create a PR. However, you will need to make a Vercel account by linking your Github then joining the Sentry team on Vercel.

File: src/api/tutorials/create-auth-token.mdx (hunk 1 above)
Comment (shanamatthews) [reply src/api/tutorials/create-auth-token.mdx:20]: Markdown doesn't care about the number, it automatically renders an ordered list with the correct numbers. https://www.markdownguide.org/basic-syntax/#ordered-lists

I prefer to use all 1s as it's much easier to add/remove steps later.

**Pull request merged**